ext/config.m4

```diff
@@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
     done
   fi
 
+#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug
```

Comment:
My previous commit was changed due to some auto generated build files. I
committed a new one with the provided commit message. Hope this does the
job here is the commit id 842915e04e521d9e62edda7187d7098ffdcefdf7
and this is the direct link
https://github.com/mohamad-z/php-driver/commit/842915e04e521d9e62edda7187d7098ffdcefdf7

On Mon, Dec 21, 2015 at 6:41 PM, Michael Penick <notifications@github.com>
wrote:

> In ext/config.m4
> <https://github.com/datastax/php-driver/pull/56#discussion_r48154135>:
>
> > @@ -217,6 +217,10 @@ if test "$PHP_CASSANDRA" != "no"; then
> >      done
> >    fi
> >
> > +#changed by mohamad zeynali mohammad.basu@gmail.com (apache license) to fix undefined reference to `deflateInit_' bug
>
> Amend would be preferred, but either way is fine.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/datastax/php-driver/pull/56/files#r48154135>.
>